sdk/ai/azure-ai-client/samples/agents/async_samples/sample_agents_basics_async.py

```diff
@@ -0,0 +1,60 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import asyncio
+import logging
+
+from azure.ai.client.aio import AzureAIClient
+from azure.identity import DefaultAzureCredential
+
+import os
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
+connection_string = os.environ["AI_CLIENT_CONNECTION_STRING"]
+
+ai_client = AzureAIClient.from_connection_string(
+    credential=DefaultAzureCredential(),
+    connection=connection_string,
+)
+
+# Or, you can create the Azure AI Client by giving all required parameters directly
+"""
+ai_client = AzureAIClient(
+    credential=DefaultAzureCredential(),
+    host_name=os.environ["AI_CLIENT_HOST_NAME"],
+    subscription_id=os.environ["AI_CLIENT_SUBSCRIPTION_ID"],
+    resource_group_name=os.environ["AI_CLIENT_RESOURCE_GROUP_NAME"],
+    workspace_name=os.environ["AI_CLIENT_WORKSPACE_NAME"],
+    logging_enable=True, # Optional. Remove this line if you don't want to show how to enable logging
+)
+"""
+async def main():
+    async with ai_client:
+        agent = await ai_client.agents.create_agent(
+            model="gpt-4-1106-preview", name="my-assistant", instructions="You are helpful assistant"
+        ) 
+        logging.info(f"Created agent, agent ID: {agent.id}")
+        
```

Comment:
Please just use "print" statements instead of "logging" to align with other samples. Are you using "logging" because you want to redirect logs to something other than the console?

Reply:
We used logging in my other branch.

That was selected instead of prints mainly I used them more, you can adjust filters etc.
We are fine to use prints also if that is the "standard" for samples

Reply:
Done